Result/ResultProtocol.swift

```diff
@@ -121,31 +121,6 @@ public extension ResultProtocol where Error: ErrorProtocolConvertible {
 
 // MARK: - Operators
 
-infix operator &&& : LogicalConjunctionPrecedence
-
-/// Returns a Result with a tuple of `left` and `right` values if both are `Success`es, or re-wrapping the error of the earlier `Failure`.
-@available(*, deprecated, renamed: "ResultProtocol.fanout(self:_:)")
-public func &&& <L: ResultProtocol, R: ResultProtocol> (left: L, right: @autoclosure () -> R) -> Result<(L.Value, R.Value), L.Error>
-	where L.Error == R.Error
-{
-	return left.fanout(right)
-}
-
-precedencegroup ChainingPrecedence {
-	associativity: left
-	higherThan: TernaryPrecedence
-}
-
-infix operator >>- : ChainingPrecedence
-
-/// Returns the result of applying `transform` to `Success`es’ values, or re-wrapping `Failure`’s errors.
-///
-/// This is a synonym for `flatMap`.
-@available(*, deprecated, renamed: "ResultProtocol.flatMap(self:_:)")
```

Comment:
The operator definitions must be remained to do that. However, we actually want to remove the operators to close #191.